library/network/test/susefirewalld_services_test.rb

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SuSEFirewallServices"
+Yast.import "SCR"
+Yast.import "FileUtils"
+
+describe Yast::SuSEFirewallServices do
+
+  before :example do
+    # We shouldn't run this test if firewalld is running
+    if Yast::SuSEFirewallServices.is_a?(Yast::SuSEFirewall2ServicesClass)
+      skip "SuSEfirewall2 backend is not supported by this test" do
```

Comment:
OK, we have clarified: the tests pass if firewalld-0.4.1.2 (on its way to factory) is used, which implements `--get-short`. (removing that(?) in lib/firewalld.rb fixes all tests)